app/talisman.py

```diff
@@ -0,0 +1,50 @@
+import copy
+import random
+import string
+
+from flask import request
+from flask_talisman import Talisman as BaseTalisman
+
+NONCE_LENGTH = 16
+
+
+class Talisman(BaseTalisman):
```

Comment:
Is this something that we could potentially contribute back to flask-talisman?

Reply:
Definitely - I didn't want to delay the implementation if we had to wait on the upstream contributions to get merged. Hopefully get this contributed back and we can remove our custom `Talisman` class.

Reply:
We should raise this sooner rather than later because after this is done we will forget.

Reply:
Fab 👍